### PR TITLE
scrambles for older years

### PR DESCRIPTION
**Pull request by guga31bb**
Still a work in progress. The older years of data are a mess in the NFL pbp

### CODE REVIEW

**Comment by guga31bb** [comment]
@mrcaseb I think this is ready except for the test failing because it changes the output and no longer matches expected output (eg scramble, rush, pass etc change on some plays)

File: R/helper_add_nflscrapr_mutations.R [hunk 1]
```diff
@@ -666,8 +666,8 @@ make_model_mutations <- function(pbp) {
 
 
 fix_scrambles <- function(pbp) {
-  # skip below code if 2005 is not in the data
-  if (!2005 %in% pbp$season) return(pbp)
+  # skip below code if <= 2005 is not in the data
+  if (min(pbp$season > 2005)) return(pbp)
```
Comment (mrcaseb) [inline R/helper_add_nflscrapr_mutations.R:670]: This computes the minimum of a logical vector instead of the minimum season.

File: NEWS.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 - Fixed a bug in `fixed_drive` and `fixed_drive_result` where the second weather delay in `2023_13_ARI_PIT` wasn't identified correctly. (#461)
 - `punter_player_id`, and `punter_player_name` are filled for blocked punt attempts. (#463)
 - Fixed an issue affecting scores of 2022 games involving a return touchdown (#466)
+- Added identification of scrambles from 1999 through 2004 with thank to Aaron Schatz
```
Comment (mrcaseb) [inline NEWS.md:14]: Could you add (#468) to this?

**Review by mrcaseb — CHANGES_REQUESTED** [review]

**Comment by mrcaseb** [comment]
I can update test expectation but will be out of town until Sunday

File: R/helper_add_nflscrapr_mutations.R (hunk 1 above)
Comment (guga31bb) [reply R/helper_add_nflscrapr_mutations.R]: Ok I think fixed

**Review by mrcaseb — APPROVED** [review]

**Comment by mrcaseb** [comment]
Seems like MacOS isn't happy with the new expectation. I will review later. 

Also I introduced some new "global variable" warnings that could be fixed in this PR as well

**Comment by guga31bb** [comment]
Tests still failing?

**Comment by tanho63** [comment]
image
rounding error, looks like. I would try adding this tolerance param to expect_equal and set it to ~ 0.00001 or so

**Comment by guga31bb** [comment]
Still fails even with a pretty lenient tolerance. I am confused. It's like it's ignoring the tolerance param.

**Review by mrcaseb — APPROVED** [review]

**Pull request merged**